shell.nix

```diff
@@ -11,8 +11,8 @@ pkgs.mkShell {
     helm-docs
     nixos-shell
     kind
-    python311
-    (poetry.override { python3 = python311; })
+    python313
+    poetry # Python3.13 is not supported (Overriding python3 input will not work)
```

Comment:
if 3.13 is not supported how will this work after this change then? poetry uses a different version itself?